Misc/NEWS.d/next/Library/2017-07-24-23-52-39.bpo-30931.mZMsJ7.rst

```diff
@@ -0,0 +1,3 @@
+asyncore: poll() and poll2() functions now check if a dispatcher was closed (if
+it's file descriptor is no more in the map) before calling its handlers:
+handlers of closed dispatchers are no more called.
```

Comment:
if it's file descriptor -> if its file descriptor
is no more in the map -> is no longer in the map?
no more called -> no longer called